src/java_bytecode/generate_java_generic_type.h

```diff
@@ -31,8 +31,4 @@ class generate_java_generic_typet
   message_handlert &message_handler;
 };
 
-void instantiate_generics(
-  message_handlert &message_handler,
-  symbol_tablet &symbol_table);
```

Comment:
How does a removal of a function fit the PR subject?

Reply:
Good point - will attempt to think of a better name for the PR.

Reply:
Could it please just be two different PRs?